packages/client/package.json.lerna_backup

```diff
@@ -0,0 +1,66 @@
+{
```

Comment:
This file does not need to be committed. You can either add it to the .gitignore or delete it.